<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractBackground</class>
 <widget class="QFrame" name="ExtractBackground">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>139</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Extract Background</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QSpinBox" name="k_neighbors">
     <property name="prefix">
      <string>K:</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="extract_n_plane">
     <property name="text">
      <string>Extract N Planes</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="extract_plane">
     <property name="text">
      <string>Extract Plane</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="unextract_plane">
     <property name="text">
      <string>Unextract Plane</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDoubleSpinBox" name="dist">
     <property name="prefix">
      <string>Distance:</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.001000000000000</double>
     </property>
     <property name="maximum">
      <double>0.500000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QToolButton" name="extract_points">
     <property name="text">
      <string>Extract Points</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QSpinBox" name="n_planes">
     <property name="prefix">
      <string>N:</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QToolButton" name="unextract_points">
     <property name="text">
      <string>Unextract Points</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QToolButton" name="undo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Undo</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
